dockerfiles/prod/Dockerfile-nginx

```diff
@@ -0,0 +1,3 @@
+FROM nginx:1.11.5
+COPY contrib/nginx.docker /etc/nginx/conf.d/default.conf
+RUN mkdir -p /opt/jarbas
```

Comment:
We don't need to use a different Dockerfile to prod and dev. Should be the same.